frontend/src/components/regions/DueTimeFormModal/Presentation.js

```diff
@@ -1,41 +1,36 @@
-import React from 'react';
-import { makeStyles } from '@material-ui/core/styles';
-import {
-  TextField,
-  Button,
-  Grid,
-  Paper,
-} from "@material-ui/core";
-import Modal from '../../widgets/Modal';
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import { TextField, Button, Grid, Paper } from "@material-ui/core";
+import Modal from "../../widgets/Modal";
 
 const useStyles = makeStyles({
-
-    dueTimeButton: {
-        marginTop: "0.3rem",
-        marginRight: "0.3rem",
-    },
-
-    paper: {
-      padding: "1rem"
-    }
+  dueTimeButton: {
+    marginTop: "0.3rem",
+    marginRight: "0.3rem",
+  },
+  paper: {
+    padding: "1rem",
+  },
+  exitIcon: {
+    position: "relative",
+    top: 0,
+    left: "200px",
+  },
 });
 
-export default ({
-  card,
-  cardId,
-  handleClose
-}) => {
+export default ({ card, cardId, handleClose }) => {
   const classes = useStyles();
   const dueTime = card.dueTime;
+
   /* dueTime is either null or something like this: "2021-06-14T09:58:28Z".
      If dueTime has a value, then we strip out the seconds, 
      hence dueTime.split('').slice(0, 16).join('') is used */
-  const defaultValue = dueTime ? dueTime.split('').slice(0, 16).join('') : "";
+  const defaultValue = dueTime ? dueTime.split("").slice(0, 16).join("") : "";
   if (cardId)
     return (
       <Modal open={true} onClose={handleClose}>
         <Paper className={classes.paper}>
-          {card ?
+          {card ? (
             <form novalidate>
```

Comment:
Hey Sbonelo. Please fix the form attribute to be `noValidate`. It's giving this error:
```
index.js:1 Warning: Invalid DOM property `novalidate`. Did you mean `noValidate`?
```